package.json

```diff
@@ -4,6 +4,7 @@
   "scripts": {
     "build": "lerna run build",
     "dist-tags": "lerna exec --concurrency 1 --no-sort --stream npm dist-tag ls",
+    "install": "lerna exec --concurrency 1 --stream -- npm install --no-fund --no-optional",
```

Comment:
I thought about adding this here as well. `install-packages` to avoid overwriting `npm install` behavior generally. 

We don't want to run this all the time. Just the one time as needed. So `npm run install-packages` when neccessary, and then `npm install` after which will do the post-install `lerna bootstrap`.

Reply:
Yeah that's what I originally had and then I read your IM message again and took you literally:

[10:12 AM] Adam Krantz
    lerna exec --concurrency 1 --stream -- npm install --no-fund --no-optional
npm install

Reply:
I will revert to my original change and push a new commit. I don't see how we can conditionally run npm install packages however